migrations/202210131714933_add_schema_comments.up.sql

```diff
@@ -0,0 +1,281 @@
+COMMENT ON COLUMN public.github_pull_requests.review_decision IS 'review decision of the pull request';
+COMMENT ON COLUMN public.github_pull_requests.state IS 'state of the pull request';
+COMMENT ON COLUMN public.github_pull_requests.title IS 'title of the pull request';
+COMMENT ON COLUMN public.github_pull_requests.updated_at IS 'timestamp of when the pull request was updated';
+COMMENT ON COLUMN public.github_pull_requests.url IS 'GitHub URL of the pull request';
+COMMENT ON COLUMN public.github_pull_requests.labels IS 'labels associated to the pull request';
+COMMENT ON COLUMN public.github_pull_requests._mergestat_synced_at IS 'timestamp when record was synced into the MergeStat database';
+
+COMMENT ON TABLE public.github_repo_info IS 'info/metadata of a GitHub repo';
+COMMENT ON COLUMN public.github_repo_info.repo_id IS 'foreign key for public.repos.id';
+COMMENT ON COLUMN public.github_repo_info.owner IS 'the user or organization that owns the repo';
+COMMENT ON COLUMN public.github_repo_info.name IS 'the name of the repo';
+COMMENT ON COLUMN public.github_repo_info.created_at IS 'timestamp of when the repo was created';
+COMMENT ON COLUMN public.github_repo_info.default_branch_name IS 'the name of the default branch for the repo';
+COMMENT ON COLUMN public.github_repo_info.description IS 'the description for the repo';
+COMMENT ON COLUMN public.github_repo_info.disk_usage IS 'the number of kilobytes on disk for the repo';
+COMMENT ON COLUMN public.github_repo_info.fork_count IS 'number of forks associated to the repo';
+COMMENT ON COLUMN public.github_repo_info.homepage_url IS 'the GitHub homepage URL for the repo';
+COMMENT ON COLUMN public.github_repo_info.is_archived IS 'boolean to determine if the repo is archived';
+COMMENT ON COLUMN public.github_repo_info.is_disabled IS 'boolean to determine if the repo is disabled';
+COMMENT ON COLUMN public.github_repo_info.is_mirror IS 'boolean to determine if the repo is a mirror';
+COMMENT ON COLUMN public.github_repo_info.is_private IS 'boolean to determine if the repo is private';
+COMMENT ON COLUMN public.github_repo_info.total_issues_count IS 'number of issues associated to the repo';
+COMMENT ON COLUMN public.github_repo_info.latest_release_author IS 'the author of the latest release in the repo';
+COMMENT ON COLUMN public.github_repo_info.latest_release_created_at IS 'timestamp of the creation of the latest release in the repo';
+COMMENT ON COLUMN public.github_repo_info.latest_release_name IS 'the name of the latest release in the repo';
+COMMENT ON COLUMN public.github_repo_info.latest_release_published_at IS 'timestamp of the publishing of the latest release in the repo';
+COMMENT ON COLUMN public.github_repo_info.license_key IS 'the licence key for the repo';
+COMMENT ON COLUMN public.github_repo_info.license_name IS 'the licence name for the repo';
+COMMENT ON COLUMN public.github_repo_info.license_nickname IS 'the licence nickname for the repo';
```

Comment:
```suggestion
COMMENT ON COLUMN public.github_repo_info.license_nickname IS 'the license nickname for the repo';
```

Reply:
I have never been able to spell that correctly 🤦